src/main/java/racingcar/utils/RacingRandomNumberGenerator.java

```diff
@@ -0,0 +1,12 @@
+package racingcar.utils;
+
+public class RacingRandomNumberGenerator implements RacingNumberGenerator{
+
+    private static final int MIN_VALUE = 0;
+    private static final int MAX_VALUE = 9;
+
+    @Override
+    public int generate() {
+        return (int) (Math.random() * (MAX_VALUE - MIN_VALUE + 1));
```

Comment:
TreadLocalRandom으로 변경완료하였습니다.

Reply:
네 맞습니다! 👍 여러 클라이언트가 들어와서 동시에 호출하면 빠르게 난수를 생성해야할테고, 그때 경합하는 상황이 발생하기 때문에 ThreadLocalRandom을 사용하는 게 좋습니다. 

Math.random()의 javadoc을 읽어보시면 좋을 것 같아요!

> This method is properly synchronized to allow correct use by more than one thread. However, if many threads need to generate pseudorandom numbers at a great rate, it may reduce contention for each thread to have its own pseudorandom-number generator.